README.md

```diff
@@ -1,5 +1,7 @@
 # CE3D2
 
+![Inspired By Lasse](https://img.shields.io/badge/inspired%20by-lasse-brightgreen.svg)
```

Comment:
Is it possible to link to your GitHub account? So clicking the badge forwards to you ;D

I've seen that this may work:
`[![Inspired By Lasse](badge-link)](link-to-your-profile)`